element_facemap/facial_behavior_estimation.py

```diff
@@ -206,45 +218,50 @@ class FacemapProcessing(dj.Computed):
     # Processing Procedure
     -> FacemapTask
     ---
-    processing_time             : datetime  # time of generation of the facemap results
-    package_version=''          : varchar(16)
+    processing_time     : datetime  # time of generation of the facemap results
+    package_version=''  : varchar(16)
     """
-    
-    # Processing only the VideoRecordings that have their Info inserted.
+
+    # Process only the VideoRecordings that have their Info inserted.
     @property
     def key_source(self):
         return FacemapTask & VideoRecording.File
 
     def make(self, key):
-        video_key = (VideoRecording.File & FacemapTask & key).fetch('KEY')
-        print('video_key:', video_key)
-        task_mode = (FacemapTask & key).fetch1('task_mode')
-
-        output_dir = (FacemapTask & key).fetch1('processing_output_dir')
+        task_mode = (FacemapTask & key).fetch1("task_mode")
 
+        output_dir = (FacemapTask & key).fetch1("processing_output_dir")
         if not output_dir:
             output_dir = FacemapTask().infer_output_dir(key, relative=True, mkdir=True)
-            print('output_dir:', output_dir)
+            print("output_dir:", output_dir)
```

Comment:
```suggestion
```

Reply:
I'll remove this line completely. The code shouldn't print any information.